OpenRA.Mods.Common/Activities/Air/HeliReturnToBase.cs

```diff
@@ -89,8 +89,8 @@ public override Activity Tick(Actor self)
 				}
 			}
 
-			var exit = dest.Info.FirstExitOrDefault(null);
-			var offset = (exit != null) ? exit.SpawnOffset : WVec.Zero;
+			var exit = dest.FirstExitOrDefault(null);
+			var offset = (exit != null) ? exit.Info.SpawnOffset : WVec.Zero;
```

Comment:
The parenthesis are unnecessary (same for the ones in `Aircraft.cs`). Not a hard blocker, though.